### DISCUSSION TITLE
Have a keymap based on just 2 Modifiers

### DISCUSSION THREAD

**Discussion by LamprosPitsillos** [xremap/xremap#610] [Ideas]
```yaml
      ALT_R-ALT_L: KEY_BACKSPACE
```

Is this possible ? The config is not crushing but i think it doesn't do anything

**Comment by k0kubun** [comment]
What you observed is correct; it's currently not supposed to work. All modifiers are expected to "chain" prefixes to a future non-modifier key, so it doesn't finalize a key press event.

We could possibly handle a modifier differently if there's a keymap with only modifiers that are currently pressed. This would complicate the implementation to some extent, but I'd be open to merging that feature if somebody steps up to file a pull request.

**Comment by hab25** [comment]
> All modifiers are expected to "chain" prefixes to a future non-modifier key

Some common software, e.g., GNOME and Intellij IDEA, can understand and use `super` both as a modifier and as a final key
